templates/quiz/quiz_create.html

```diff
@@ -1,48 +1,39 @@
 {% extends 'base/base.html' %}
 
 {% block content %}
-<div class="container">
-    <form id="quiz_create_form" method="POST">
-        {{ form.csrf_token }}
-        <div class="mb-3">
-            <label for="titleInput">{{form.title.label}}</label>
-            {{form.title(class="form-control", id="titleInput")}}
-            <div id="titleHelp" class="form-text">Enter you quiz title</div>
-        </div>
-
-        <div class="mb-3">
-            {{form.category(class="form-select")}}
-        </div>
-
-
-        <div class="mb-3">
-            <label for="descInput">{{form.description.label}}</label>
-            {{form.description(class="form-control", id="descInput")}}
-            <div id="usernameHelp" class="form-text">Short description of your quiz</div>
-        </div>
-
-        <div class="mt-3">
-            <h3>Questions: </h3>
-            <div id="question-container">
+<div class="bg-image" align="center">
+    <div class="container card border-0 shadow mt-5" align="left">
+        <form id="quiz_create_form" method="POST">
+            {{ form.csrf_token }}
+            <div class="mb-3">
+                <label for="titleInput">{{form.title.label}}</label>
+                {{form.title(class="form-control", id="titleInput")}}
+                <div id="titleHelp" class="form-text">Enter your quiz title here</div>
+            </div>
+            <!--
+            <div class="mb-3">
+                {{form.category(class="form-select")}}
```

Comment:
Why this has been commented out?

Reply:
I was playing with labels and their descriptions in terms of arrangement and didn't know where to put it yet, but i'll uncomment it since it's needed for the proper form fields connection to the backend :x